NetKAN/ScattererOPM.netkan

```diff
@@ -0,0 +1,18 @@
+identifier: ScattererOPM
+name: Scatterer Config for Outer Planets Mod
+$kref: '#/ckan/github/AnisDjerrab/Scatterer-OPM'
+license: MIT
+ksp_version_min: 1.12.1
+ksp_version_max: 1.12.5
+resources:
+  homepage: https://forum.kerbalspaceprogram.com/topic/227158-scatterer-for-opm/
```

Comment:
Hi @AnisDjerrab, one last suggestion:

It would help your non-CKAN users if they could find this link from the GitHub repo itself. Luckily if you click the gear icon in the About section, there's a website field in the popup where you can paste this link. If you do that, CKAN can pull from that field instead of hard-coding it here in the .netkan file. This has the advantage of leaving the link fully under your control, so if you want to change it at any point, you don't even need to submit a pull request, you could just change it in your repo and it would automatically update in CKAN.